src/interpreter.rs

```diff
@@ -2,6 +2,35 @@ use rustpython_vm::{Interpreter, Settings, VirtualMachine};
 
 pub type InitHook = Box<dyn FnOnce(&mut VirtualMachine)>;
 
+/// The convenient way to create `rustpython_vm::Interpreter` with stdlib and other stuffs.
+/// 
+/// Basic usage:
+/// ```
+/// let interpreter = rustpython::InterpreterConfig::new()
+///     .init_stdlib()
+///     .interpreter();
+/// ```
+/// 
+/// Overriding Settings:
```

Comment:
You may want to add a link to the `Settings` struct so that downstream users may know how to change the settings to their liking.